### ISSUE TITLE
Unit test test_angle_between_planes.praat fails randomly

### ISSUE THREAD

**Issue by rlaboiss** [praat/praat#486]
The unit test  dwtest/test_angle_between_planes.praat fails randomly:

```
$ while true ; do xvfb-run praat --run dwtest/test_angle_between_planes.praat ; done
test_angle_between_planes.praat
test_angle_between_planes.praat OK
test_angle_between_planes.praat
test_angle_between_planes.praat OK
[...]
Error: Script assertion fails in line 31 (false):
   abs(degrees-degrees_found) < 2e-6; 280 0 2.091309789151873e-06
Script “/var/data/rlaboiss/debian/PKGS/praat/praat/dwtest/test_angle_between_planes.praat” not completed.
Praat: script command <<dwtest/test_angle_between_planes.praat>> not completed.
```
The problem seems to occur when the rotation angle is equal to zero.  I did not investigate this issue deeply, but changing the line:

```
        hoek = randomInteger (0,90) 
```

into:

```
        hoek = randomInteger (1,90) 
```

solved the problem for me.

**Comment by davidweenink** [comment]
Yes 
hoek = randomUniform (0.001, 90) 
solves the issue. For small angles we probably need better solution.

**Comment by rlaboiss** [comment]
Thanks, David. Just for the record, this is the commit that fixed the problem: 317fee05a186ef46f8de7f49ae82b447155eeab5
